packages/shared/src/hooks/notifications/useBookmarkReminder.ts

```diff
@@ -27,24 +28,26 @@ interface BookmarkReminderProps {
 }
 
 interface UseBookmarkReminder {
-  onBookmarkReminder: (props: BookmarkReminderProps) => void;
+  onBookmarkReminder: (props: BookmarkReminderProps) => Promise<EmptyResponse>;
 }
 
 export const getRemindAt = (
   date: Date,
   preference: ReminderPreference,
 ): Date => {
+  const atNineAM = (value: Date) => set(value, { hours: 9, minutes: 0 });
```

Comment:
shall we modify the tests for this change?

Reply:
Yep. Should be part of the tests to ensure accurate result.

Reply:
Done ✅